package.json

```
@@ -11,14 +11,14 @@
     "lodash": "^4.17.4",
     "make-promises-safe": "^1.1.0",
     "mocha": "^4.0.1",
+    "semantic-release": "^8.2.0",
     "semver": "^5.4.1",
-    "standard": "^10.0.3",
-    "standard-markdown": "^4.0.2",
-    "semantic-release": "^8.2.0"
+    "standard": "^10.0.3"
   },
   "scripts": {
+    "update": "./script/update.sh",
     "build": "node script/build.js",
-    "test": "mocha && standard --fix && standard-markdown",
+    "test": "mocha && standard --fix",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
```

so many for semantic release, we can bump this to the latest and leave this to a single line, how in others modules?
```suggestion
    "semantic-release": "semantic-release"
```

---

Yes, I noticed this was different than the other projects too. I'm not sure why it's different, but I didn't change it. Happy to try it and see if it works. :)

---

> Happy to try it and see if it works. :)

Let's go on wheels of experiments and tries

---

This might be because our version of semantic-release is very old. I'll bump it as part of this PR.